src/runvm-coreos-installer

```diff
@@ -58,3 +68,7 @@ mount --bind "${workdir}" "${deploydir}/${workdir}"
 set -x
 
 RUST_BACKTRACE=full chroot "${deploydir}" env -C "${workdir}" coreos-installer "$@"
+
+if [ -f /var/tmp/coreos-installer-output ]; then
+    cp /var/tmp/coreos-installer-output /dev/virtio-ports/coreosout
+fi
```

Comment:
It's declared as an API in the header. It's similar to how `/dev/disk/by-id/virtio-coreos` works too.

It's tricky because this script needs to remain agnostic to what coreos-installer command is actually run. So we have to setup generic APIs like this.

Reply:
The actual switches that the caller passes depends on the command used. All this is doing is declaring an API for passing a file back to the host. The caller can use it as they'd like.